SA/project_documentation/content/used_tools.typ

```diff
@@ -0,0 +1,22 @@
+= Used Tools
+
+We used these tools for the realization of this project.
+
+/ Jira: is a software management tool that can be used to manage agile teams.
+  We are using it to plan the project and keep track of the progress.
+  #footnote("https://www.atlassian.com/software/jira")
```

Comment:
```suggestion
  #footnote("https://visual-fp-ost.atlassian.net/jira/")
```

Reply:
I didn't want to add links to our specific projects, since our instances will probably be teared down at some point, but this document will probably be released to the public

Reply:
It's more of a convenience things for Farhad anyway. I don't think that it will have any influcen on grading. If you want you can leave it as it is